openapi/openapi.yaml

```diff
@@ -1287,16 +1292,31 @@ components:
           type: string
       required: [title, url]
 
-
-    User:
+    UserMetadataInDB:
```

Comment:
`UserMetadataInDB` is for internal use only so I think it may be a bit confusing if it is defined in `openapi.yaml`. This file should only include public interfaces that are intended for the users to see. Also, a dataclass is usually easier to work with than swagger `Model` types. If we want to store more attributes in the User collection but we don't expose that information to the clients, we can just update that dataclass. If it is defined as a public interface, we need to release a new API version whenever we update it (and we may potentially introduce breaking changes for the clients).

Reply:
Thanks for pointing this out! I've reflected it.